main/mainTemplate.json

```diff
@@ -64,12 +64,12 @@
         "description": "Indicates whether the Public IP is new or existing."
       }
     },
-    "securityGroupBootstrapUi": {
-      "type": "string",
-      "defaultValue": "false",
+    "securityGroup": {
+      "type": "array",
+      "defaultValue": "bootstrapUi",
       "allowedValues": [
-        "false",
-        "true"
+        "bootstrapUi",
+        "ssh"
```

Comment:
we are not allowing SSH port. do we need to mention in allowed values ?

Reply:
Then we don't add ssh?

Reply:
yes not required

Reply:
Hi @nnallamo ,
This is just for QA version image.